network.go

```diff
@@ -469,3 +469,10 @@ func (n *network) getSvcRecords() []etchosts.Record {
 
 	return recs
 }
+
+func (n *network) scope() (driverapi.Scope, error) {
```

Comment:
If we define `DataScope()` function in `datastore.KV`, should we also move `type DataScope` from driverapi.go to datastore.go?